### DISCUSSION TITLE
After last update is no more possible training on <8gb?

### DISCUSSION THREAD

**Discussion by Nyaster** [d8ahazard/sd_dreambooth_extension#663] [Q&A] [answered]
After last update is no more possible training on <8gb? Before update all working and consuming around 6gb, now always is cuda out of memory, in the same settings/

**Comment by Boldor83** [comment, marked as answer]
yes even lora doens't work anymore :( before it used less than 5 GB. I have 8 GB vram. It tries to allocate the same vram wether i check lora or not.  Both times: CUDA out of memory. Tried to allocate 20.00 MiB (GPU 0; 8.00 GiB total capacity; 7.14 GiB already allocated; 0 bytes free; 7.28 GiB reserved in total by PyTorch)

**Reply by andykaufseo** [reply]
can confirm this, could use a batch of 4 with DB and 12 with lora, now can't do more than 4 with lora and 1 with regular DB.

hopefully @d8ahazard will confirm what's going on and fix

**Reply by iznanka** [reply]
yes, in order to start learning again on 8gb, you need to install a separate web-gui for Lora https://github.com/cloneofsimo/lora/discussions/102
